plugins/field-time/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "blockly-plugin-field-time",
+  "version": "0.0.0",
+  "description": "A Blockly plugin.",
+  "scripts": {
+    "audit:fix": "blockly-scripts auditFix",
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "lint": "blockly-scripts lint",
+    "predeploy": "blockly-scripts predeploy",
+    "prepublishOnly": "npm run clean && npm run build",
+    "start": "blockly-scripts start",
+    "test": "blockly-scripts test"
+  },
+  "main": "./dist/index.js",
+  "module": "./src/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "",
+  "keywords": [
+    "blockly",
+    "blockly-plugin",
+    "field-time"
+  ],
+  "homepage": "https://github.com/team-vibe-labs/blockly-samples/tree/master/plugins\\field-time#readme",
+  "bugs": {
+    "url": "https://github.com/team-vibe-labs/blockly-samples/issues"
+  },
```

These will need to be updated as well:
```suggestion
  "homepage": "https://github.com/google/blockly-samples/tree/master/field-time#readme",
  "bugs": {
    "url": "https://github.com/google/blockly-samples/issues"
  },
```